content/manuals/desktop/release-notes.md

```diff
@@ -66,6 +66,13 @@ Take a look at the [Docker Public Roadmap](https://github.com/orgs/docker/projec
 
 - Kubernetes cluster may not start if **Registry Access Manager** is enabled. As a workaround, add `registry.k8s.io` and `<geo>-docker.pkg.dev` to **Registry Access Management** policies.
 
+### Deprecation
+
+#### For Mac
+
+- QEMU (Legacy) as a VMM on Apple Silicon will be removed in 4.39. Please switch to virtualization.framework for increased performance and stability. If you encounter an issue, [contact support](https://www.docker.com/support/) or file a [github issue](https://github.com/docker/for-mac/issues).
```

Comment:
- Looks like Vale doesn't like the "Please"; perhaps "Users are recommended to switch to ..."
- It also complains about "VMM" as unknown acronym; I think it works if you put it  like "Virtual Machine Manager (VMM) ..." (is it "manager"?)